unlock-protocol-com/blog/grants-program-updates.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Unlock Announces Grants Program Upgrade
+subTitle: Improvements go live to streamline processes and accelerate growth for the Unlock ecosystem
+description: Improvements go live to streamline processes and accelerate growth for the Unlock ecosystem.
+author: Angela Steffens
+date: November, 1 2022
+image: /images/blog/grants-upgrade-share.png
+---
+
+## Improvements  go live to streamline processes and accelerate growth for the Unlock ecosystem.
```

Comment:
line 10 is not needed, redundant with the description / subTitle above. please remove.

Reply:
Removed